### ISSUE TITLE
Magento does not send an email about a refunded grouped product

### ISSUE THREAD

**Issue by atwixfirster** [magento/magento2#26245]
### Preconditions (*)

1. Create an order with a **grouped** product
2. Create an invoice for your order

### Steps to reproduce (*)

1. Create a Credit Memo for your order and check "**Email Copy of Credit Memo**" option
2. Press "Refund Offline" button

<img width="1553" alt="26245_02" src="https://user-images.githubusercontent.com/13585327/71698746-4228d180-2dc5-11ea-9434-bb837e918a5f.png">

### Expected result (*)

1. Email is sent to customer

### Actual result (*)

<img width="1354" alt="26245_01" src="https://user-images.githubusercontent.com/13585327/71698776-61276380-2dc5-11ea-9270-6d289ebe36d9.png">

PHP error:
```
Fatal error: Uncaught TypeError: Argument 1 passed to Magento\Sales\Block\Order\Email\Items\Order\DefaultOrder::getItemPrice() must be an instance of Magento\Sales\Model\Order\Item, instance of Magento\Sales\Model\Order\Creditmemo\Item given, called in /home/dev/sites/magento2/app/code/Magento/Sales/view/frontend/templates/email/items/creditmemo/default.phtml on line 32 and defined in /home/dev/sites/magento2/app/code/Magento/Sales/Block/Order/Email/Items/Order/DefaultOrder.php:99 Stack trace: #0 /home/dev/sites/magento2/app/code/Magento/Sales/view/frontend/templates/email/items/creditmemo/default.phtml(32): Magento\Sales\Block\Order\Email\Items\Order\DefaultOrder->getItemPrice(Object(Magento\Sales\Model\Order\Creditmemo\Item)) #1 /home/dev/sites/magento2/lib/internal/Magento/Framework/View/TemplateEngine/Php.php(75): include('/home/dev/sites...') #2 /home/dev/sites/magento2/lib/internal/Magento/Framework/View/Element/Template.php(272): Magento\Framework\View\TemplateEngine\Php->render(Object(Magento\Groupe in /home/dev/sites/magento2/app/code/Magento/Sales/Block/Order/Email/Items/Order/DefaultOrder.php on line 99
```

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @dmytro-ch 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-30185` were created

**Issue Available**: @dmytro-ch, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @atwixfirster. Thank you for your report. 
The issue has been fixed in magento/magento2#26246 by @atwixfirster in 2.4-develop branch
Related commit(s):
 - [7141278628d4ef0d54d244ab337cbea62e7f37d8](https://github.com/magento/magento2/commit/7141278628d4ef0d54d244ab337cbea62e7f37d8)
 - [a5f0d30e309aa8d2db9f138254cb1d35c752ec99](https://github.com/magento/magento2/commit/a5f0d30e309aa8d2db9f138254cb1d35c752ec99)
 - [b1d99e54dd9d9ca0881db11c8e9ca6a04a50b268](https://github.com/magento/magento2/commit/b1d99e54dd9d9ca0881db11c8e9ca6a04a50b268)
 - [dfc44a1288d36f2a0a894958c7caecd8dc594c46](https://github.com/magento/magento2/commit/dfc44a1288d36f2a0a894958c7caecd8dc594c46)

The fix will be available with the upcoming 2.4.0 release.
